client/cypress/e2e/navigation.cy.ts

```diff
@@ -4,8 +4,7 @@ describe('navigation', () => {
   });
 
   it('access to project detail page and switch dashboard', () => {
-    cy.visit('/');
-    cy.get('a[data-cy="project-item-link"]').first().click();
+    cy.visit('/projects/indonesia-landscape');
```

Comment:
This should not work as now we have project_code on the url. Should be something like:
`projects/AFoCO_036_2022`